interfaces/filter-effects.idl

```diff
@@ -0,0 +1,21 @@
+interface SVGFETurbulenceElement : SVGElement {
```

Comment:
https://drafts.fxtf.org/filter-effects/ has a cartload of IDL, so something gone awry here.

Reply:
Yep, this was a bug in the FOAM model which I switched too. PTAL.